src/weather.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable import/no-cycle, import/no-cycle, import/prefer-default-export */
```

Comment:
Kindly remote comment and avoid the inline ESLint disabling rule, this is equally the same as altering or changing our linter files. Let us fix it